index.html

```diff
@@ -556,6 +560,8 @@
           </li>
           <li>Let <var>request</var> be a new <a>PaymentRequest</a>.
           </li>
+          <li>Set <var>request</var>.<a>[[\paymentRequestID]]</a> to <var>paymentRequestID</var>.
```

Comment:
OK, so now it is stored with the object. But it is never used by the entire spec, right? You need to wire up the getter for the paymentRequestID property to return the value of the [[paymentRequestID]] internal slot.